.bazelrc

```diff
@@ -194,3 +202,15 @@ build:windows --per_file_copt='external/v8/src/objects/swiss-name-dictionary\.cc
 # enable clang coverage: https://clang.llvm.org/docs/SourceBasedCodeCoverage.html
 build:clang-coverage --copt="-fprofile-instr-generate" --linkopt="-fprofile-instr-generate"
 build:clang-coverage --copt="-fcoverage-mapping" --linkopt="-fcoverage-mapping"
+
+# Debug
```

Comment:
Usually debug builds are less intense than ASAN.

Reply:
macOS failure is a timeout: //src/workerd/api:sql-test after ~300s, test completes in 25s on an uncontested M1.

Windows build failure does not repro on Windows dev machine. It looks like it's falling fowl of the hackery that was needed for LLVM16 and is shortly going away.

Reply:
I half suspect the macOS issue is due to invoking llvm-symbolizer and it doing a crazy slow symbol search just based on watching top whilst bazel runs the test (detached from the terminal).